draft-ietf-bidding-and-auction-services.md

```diff
@@ -906,6 +918,27 @@ response from Bidding and Auction Services. It takes as input the
        `included_groups[owner]`, return failure.
     1. Let `name` be the `interest group name` for `included_groups[owner][element]`.
     1. Append the tuple (`owner`, `name`) to `processed response["bidding groups"]`.
+1. If `response["updateGroups"]` exists and is a map:
+  1. For each `key`, `value` in `response["updateGroups"]`:
+    1. Let `owner` be equal to `key` parsed as an [ORIGIN], continuing the next
+      iteration of this loop if there is an error.
+    1. If `request context`'s `included_groups` does not contain `owner` as a
+       key, continue the next iteration of this loop.
+    1. If `value` is not a list, return failure.
+    1. For each `element` in `value`:
+      1. If `element` is not a map, continue the next iteration of this loop.
+      1. If `element["index"]` does not exist or is not an integer or
+         `element["time"]` does not exist or is not an integer, continue the
+         next iteration of this loop.
+      1. If `element["index"]` is not an integer or `element["index"] < 0`, return failure.

```

Comment:
fine as-is, but why fail and not just continue?

Reply:
That's a copypasta bug. Fixed.